value/bucket.go

```diff
@@ -52,9 +55,31 @@ func (b *BucketBlueprint) String() string {
 		evictionPolicy = b.EvictionPolicy
 	}
 
+	piTRGranularity := "N/A"
+
+	if b.PiTREnabled {
+		if b.PiTRGranularity != 0 {
+			piTRGranularity = strconv.FormatUint(b.PiTRGranularity, 10)
+		} else {
```

Comment:
nit: could avoid the 'else' by doing

pitrGranularity = "default"
if b.PiTRGranularity != 0 {
    pitrGranularity = ...
}

IIRC we discussed this on a review a while ago and found a style guide which recommended doing it this way.

Reply:
Done.